### ISSUE TITLE
Using proxy.page.set with PhantomJS 1.8.0 always returns false even if the operation succeeds

### ISSUE THREAD

**Issue by jonbuffington** [sheebz/phantom-proxy#11]
Using _proxy.page.set_ with PhantomJS 1.8.0 always returns false to the callback function even if the operation succeeds. The debug output does not indicate any problems.

A source snippet is:

``` javascript
function(proxy, fn) {
  proxy.page.set('paperSize', {format: 'Letter', orientation: 'portrait', margin: '0.25in'}, function(result) {
    var err = result ? null : new Error('Unable to set paperSize.');
    fn(err, proxy);
  });
},
```

The debug output snippet is:

```
creating proxy to http://localhost:1061
creating phantom proxy instance with options {"debug":true,"port":1061,"host":"localhost","hostAndPort":"http://localhost:1061","eventStreamPath":"/Users/jcb/Projects/Resonate/rendering/src/main/js/node_modules/phantom-proxy/temp/events.txt","clientPort":61290}
pinged server and got back false
creating a new server, waiting for servercreated event on [object Object]
opening control page
success
connection
response: {"type":"serverCreated","args":[true],"source":"server"}
buffer :{"type":"serverCreated","args":[true],"source":"server"}
emitting serverCreated event on server
server created callback fired with value:true
calling url: http://localhost:1061/page/functions/open
```

**Comment by AndrewRayCode** [comment]
I can verify this on 1.8. the call returns nothing for body either, it is empty.

**Comment by AndrewRayCode** [comment]
addressed in https://github.com/sheebz/phantom-proxy/pull/28

**Comment by jonbuffington** [comment]
@sheebz I just tested using phantom-proxy@0.1.791 and this issue is still open. Unfortunately, I have not had time to track down the root cause yet.

**Comment by sheebz** [comment]
Looks like @DelvarWorld s change here got clobbered in merge.  going to try reapplying.

**Comment by sheebz** [comment]
@jonbuffington can you check if this fixes the problem?

**Comment by jonbuffington** [comment]
@sheebz Thanks for re-opening! This issue is resolved in phantom-proxy@0.1.791.

**Comment by sheebz** [comment]
No problem
